app/controllers/jwt_api_entreprise_controller.rb

```diff
@@ -0,0 +1,16 @@
+class JwtAPIEntrepriseController < AuthenticatedUsersController
+  before_action { authorize_user_resource_access!(user_resource_id) }
```

Comment:
J'ai rien dit sur la PR d'avant sur le fait de voir son profil sur le `users#show` parce que je me suis dit " why not ", mais là ça devient un peu tiré par les cheveux avec les nested je trouve.

Je simplifirai tout ça en omettant l'id du user dans la route, qui est déjà dans la session (avec un namespace du genre `profile`), et je ferai directement dans ce contrôller `current_user. jwt_api_entreprise`.

Ça donnerait en terme de routes (tbd, je propose ça vu que je fais ça la plupart de mes apps).

```
/profile => le show du current_user
/profile/tokens => les jetons du current user
/profile/tokens/:id => le jeton du current user
```

C'est plus propre, et ça simplifira les traitements (en gros -> on flip tous les `before_action` là)

Reply:
Ok je peux faire ça. 

J'ai repris l'existant où l'URL est identique pour les simples users et admins d'où l'ID dans les URLs.